zaza/openstack/utilities/openstack.py

```diff
@@ -1690,6 +1690,30 @@ def delete_image(glance, img_id):
     delete_resource(glance.images, img_id, msg="glance image")
 
 
+@tenacity.retry(wait=tenacity.wait_exponential(multiplier=1, max=60),
```

Comment:
resource_removed called from delete_resource is already using tenacity to retry the operation.  Why is the extra level of retrying necessary?

Reply:
removed